_data/en/settings.yml

```diff
@@ -129,8 +126,7 @@ help_page:
   p_1: Browse common topics
   title: How can we help?
   categories:
-    - title: Get started with login.gov
-      description: Create your account. Learn about authentication options and more account features.
+    - title: Get started with Login.gov       description: Create your account. Learn about authentication options and more account features.
```

Comment:
Looks like this was accidentally folded onto the preceding line. There's quite a few of these issues throughout the data files.

```suggestion
    - title: Get started with Login.gov
      description: Create your account. Learn about authentication options and more account features.
```